extensions/amp-truncate-text/0.1/amp-truncate-text.css

```diff
@@ -0,0 +1,53 @@
+/**
+ * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+amp-truncate-text {
+  padding: 0 !important;
+  border: none !important;
+}
+
+amp-truncate-text[i-amphtml-truncate-expanded] {
+  height: auto !important;
+}
+
+amp-truncate-text[i-amphtml-truncate-expanded] .i-amphtml-truncate-content {
+  position: static;
```

Comment:
This actually seems like should be `!important`?

Reply:
So far we've been very protective over internal elements, essentially using `!important` a lot. Obviously in SD case that's a mute point. Overall, I'm usually very protective over the `position` CSS specifically - I've seen a lot of attempts to "hack" it around.

Reply:
Sounds good. Changed everything in the non-shadow version to use `!important` for things that should not be changed. The shadow version should already have `!important` on the host element and anything that is slotted.